src/local/delete_test.go

```diff
@@ -63,16 +63,29 @@ func TestDeleteEmptyDirectories(t *testing.T) {
 	assert.NoError(t, err)
 	manager := NewManager(logger, m, components.NewProvider(nil))
 
-	// Create empty hidden dir
-	assert.NoError(t, os.MkdirAll(filepath.Join(projectDir, `.hidden`), 0755))
-	// Create empty sub-dir in hidden dir
+	// Structure:
+	// D .hidden
+	// D .git
+	//     D empty
+	// D tracked-empty
+	// D tracked-empty-sub
+	//     D abc
+	// D non-tracked-empty
+	// D tracked
+	//    F foo.txt
+	// D non-tracked
+	//    F foo.txt
+	// D tracked-with-hidden
```

Comment:
Chyba sa tyka mazania prazdnych adresarov.

Podarilo sa mi nasimulovat problem, ktory nahlasil Vlado.
Vid v teste: `tracked-with-hidden -> .git`.

Jedna sa o "prazdny" adresar, ... ktory ma v sebe ignorovany adresar, napr. `.git`.
V tom pripade sme `.git` ignorovali a chybne urcili, ze nadradeny adresar je prazdny.

Ked sa nasledne tento "prazdny" adresar mazal, tak nastala chyba `directory not empty`.
![image](https://user-images.githubusercontent.com/19371734/132907988-f39c9739-7ba9-468a-8648-b4d61c9a05ec.png)

V prvom commite je pridany padajuci test, a v druhom je fix.